akka-docs/src/test/scala/docs/actor/ActorDocSpec.scala

```diff
@@ -5,21 +5,29 @@ package docs.actor
 
 import jdocs.actor.ImmutableMessage
 
+//#fiddle_code
 import language.postfixOps
+//#fiddle_code
 
+//#fiddle_code
```

Comment:
I'm not sure how paradox handles these nested inclusion tags, could you move the needed imports down to above the `case object Ping`, so that the example becomes more self contained